routes/project.js

```diff
@@ -5,10 +5,23 @@ var modes = require('js-git/lib/modes');
 var fsdb = require('git-node-fs/mixins/fs-db');
 var createMixin = require('js-git/mixins/create-tree');
 var formatsMixin = require('js-git/mixins/formats');
+var passport = require('passport');
+var WindowsAuthentication = require('passport-windowsauth');
+var access = require('access-check');
 //var async = require('async');
 var router = express.Router({ mergeParams: true });
 
 var base;
+var users = {};
+
+passport.use(new WindowsAuthentication({integrated: true}, function(profile, done) {	
+	var user = users[profile.id];   
+    if (!user) {
+    	profile.email = profile.id + '@uss.co.uk';
+    	user =  profile;    	
+    	done(null, user);
+    }
+}));
```

Comment:
Remove this.. and the windowsAuthentication require. This is all done in apps.js